### ISSUE TITLE
Set up additional buttons by configuration

### ISSUE THREAD

**Issue by avzhuiko** [avzhuiko/climate-remote-control#19]
Usually AC has additional features which doesn't have a state. Ex: Turbo mode, Ionisation, sleep mode.
Also these features can be have state. And good to have toggle mode or select for them

**Comment by hunterdrayman** [comment]
Good idea! On my AKB74955603 remote control there is a (stateless) "Light off" button. This toggles the illuminated display on my AC09BK NSJ A/C unit, for use in a dark room when sleeping. I have found I can make use of your Preset feature to invoke it but then toggling it back again is tricky so an actual toggle button would be nice.
